operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -570,7 +569,6 @@ func TestInstallPackagePath(t *testing.T) {
 	if err := tgz.Create(string(liveCharts), filepath.Join(serverDir, testTGZFilename)); err != nil {
```

Comment:
we can drop this now and delete the tgz.Create method

Reply:
Now that tgz.Extract is only used for test maybe we just replace it with a simple `exec(tar ....)`. Not sure, its just a bit complex to maintain only for a test..